gulp/util.js

```diff
@@ -48,6 +49,9 @@ function buildJs () {
 
   var jsBuildStream = gulp.src( jsFiles )
       .pipe(filterNonCodeFiles())
+      .pipe(babel({
+          presets: [["env", {targets: {browsers: config.browsers}}]]
```

Comment:
If I understand it correctly `useBuiltIns` alone won't solve the minification issues. I'm not sure what uglify + `useBuiltIns` does.

The option `useBuiltIns`seems to require a module loader (see [explanation] (https://github.com/babel/babel-preset-env/releases/tag/v1.0.0)).